### ISSUE TITLE
 Add Github link with icon in Footer

### ISSUE THREAD

**Issue by sowmya-1313** [PratikMane0112/TelMedSphere#22]
### Description

Description:
I like to enhance our website by adding social media links to the footer. This will help users easily find and follow our social media profiles. The social media links should include Facebook, Twitter, LinkedIn, and Instagram by which  Each icon will  be clickable and link to our respective social media profiles. using HTML CSS.

### Screenshots

![social links](https://github.com/user-attachments/assets/a49bdb64-c052-4dab-b5b5-30f0809b37a1)

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/PratikMane0112/TelMedSphere/blob/master/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a SWOC'25 contributor.
- [ ] I am a KWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/PratikMane0112/TelMedSphere/blob/master/.github/CODE_OF_CONDUCT.md)".

**Comment by PratikMane0112** [comment]
Hey @sowmya-1313, Welcome to 💖TelMedSphere !!!! Thanks for opening an issue, please go ahead. For any query or updates join discord (link in discription itself) Happy Coding!! ✨

**Comment by PratikMane0112** [comment]
Note : You can start now, but as SWoC guidelines I can merge your PR only after 1 Jan.

**Comment by IceHawk11** [comment]
Hi......I'm currently from SWOC 5.0 and I wanna know on which page the social media links get put on the footer??

**Comment by PratikMane0112** [comment]
Hey @IceHawk11, WelCome to 💖 TelMedSphere !!!! Thank you for your interest. Keep patience for 2 days, let give time to @sowmya-1313, if not creted then issue will get assign to you.

**Comment by IceHawk11** [comment]
Okay sure

**Comment by sowmya-1313** [comment]
I'm sorry for any inconvenience, but due to some technical  issues in my system, I'm unable to complete the task on time. Please disassign me from this task and assign it to someone else who is interested.

**Comment by PratikMane0112** [comment]
@IceHawk11, start working on the issue, rather to adding social media icon just add github icon, which will render to this repo. As this website has no social media account tjhat links doesnt make sence

**Comment by IceHawk11** [comment]
Okay on it...at the footer of the main landing page right??

**Comment by PratikMane0112** [comment]
> Okay on it...at the footer of the main landing page right??

Yes, for any query further. Join our discord server (link in README.md itself)

**Comment by payalgupta25** [comment]
i can add the icon and link in footer, please assign it to me

**Comment by PratikMane0112** [comment]
> i can add the icon and link in footer, please assign it to me @PratikMane0112

Hey @payalgupta25, thank you for your interest but it is already assign. Feel free explore any other issue or create one .Thanks !!!!

**Comment by PratikMane0112** [comment]
@payalgupta25 Do you know tailwind ? If yes then there is bunch of things in this project to do with. Refer issue #2
